### DISCUSSION TITLE
📖 Request for Information on Cross-Chain Reentrancy and Read-Only Reentrancy

### DISCUSSION THREAD

**Discussion by 0xrisec** [pcaversaccio/reentrancy-attacks#10] [Q&A] [answered]
Hello, I noticed that you have mentioned `Cross-Chain Reentrancy` and `Read-Only Reentrancy` as types of Reentrancy attacks. However, despite searching extensively on the internet, I was unable to find any resources related to them. Could you please provide me with any relevant information?

**Comment by pcaversaccio** [comment, marked as answer]
@ROOTBABU those are rather new attacks and therefore not yet well documented. Here are some links that will be helpful:

- *Read-Only Reentrancy:*
  - https://chainsecurity.com/heartbreaks-curve-lp-oracles
  - https://chainsecurity.com/curve-lp-oracle-manipulation-post-mortem
  - https://archive.devcon.org/archive/watch/6/read-only-reentrancy-a-novel-vulnerability-class-responsible-for-100m-funds-at-risk/?tab=YouTube
  - https://forum.makerdao.com/t/curve-lp-token-oracle-manipulation-vulnerability-emergency-response-postmortem/18011
  - https://forum.makerdao.com/t/curve-lp-token-oracle-manipulation-vulnerability-technical-postmortem/18009
- *Cross-Chain Reentrancy:*
  - https://medium.com/@mateocesaroni_11308/cross-chain-re-entrancy-54ec2e924e9c

**Reply by 0xrisec** [reply]
I'm extremely grateful for the resources you provided, especially the one on Cross-Chain Reentrancy - thank you so much!
